app/controllers/ctc/questions/dependents/claim_child_anyway_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Ctc
+  module Questions
+    module Dependents
+      class ClaimChildAnywayController < BaseDependentController
+        include AuthenticatedCtcClientConcern
+        layout "yes_no_question"
+
+        def self.show?(dependent)
+          return false unless dependent
+          dependent.qualifying_child_relationship? &&
+            dependent.meets_qc_age_condition_2020? &&
+            dependent.meets_qc_misc_conditions? &&
+            dependent.meets_qc_residence_condition_2020? &&
+            dependent.can_be_claimed_by_other_yes?
```

Comment:
If we're right about the way yes/no buttons work on the previous page, then the wrong data is stored at the time you visit this page.